.github/workflows/release-tarball.yaml

```diff
@@ -0,0 +1,49 @@
+name: Release CI
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  pull_request:
```

Comment:
Can we remove this? I don't think we need to run release on any PR. Actually the release should really just be run either manually or scheduled